static/js/click_handlers.js

```diff
@@ -422,6 +422,16 @@ export function initialize() {
 
```

Comment:
@timabbott I am not sure why we see the bug I am seeing the bug I described in the commit message. Will took into it tomorrow.

Reply:
That's because you don't have `e.preventDefault()`, and the default action for `Enter` is to produce a new character.

Generally, we want all of our click handlers to be using both `preventDefault` and `stopPropagation`, and we should probably figure out how to do a systematic sweep for this category of issue, since there's a lot of copy-pasted code not doing it right.  (Or figure out how to use new code patterns, e.g. a `register_click_handler` function).

Reply:
Yeah, that was my first thought too, but doesn't look like `preventDefault ` makes a difference here.

Reply:
It was indeed `preventDefault` I was just using it in the wrong place.